docker-compose.yml

```diff
@@ -16,16 +16,18 @@ services:
       POSTGRES_PASSWORD: password
     ports:
       - "5432:5432"
+    volumes:
+      - db_data:/var/lib/postgresql/data
   graphql-engine:
     image: hasura/graphql-engine:v2.42.0
     ports:
       - "8080:8080"
     restart: always
     environment:
       ## postgres database to store Hasura metadata
-      HASURA_GRAPHQL_METADATA_DATABASE_URL: postgres://kiyent:password@host.docker.internal:5432/postgres
+      HASURA_GRAPHQL_METADATA_DATABASE_URL: postgres://kiyent:password@db:5432/MiniprojDB
```

Comment:
Upon checking it's okay to use the same database as the project's data, since Hasura creates separate tables for the metadata anyway